setup/lti-config.json

```diff
@@ -31,13 +31,14 @@
       "term_name": "$Canvas.term.name",
       "canvas_url": "$Canvas.api.baseUrl",
       "term_start": "$Canvas.term.startAt",
-      "redirect_url": "https://dev2.dev.umgpt.umich.edu/",
+      "redirect_url": "https://main.dev.umgpt.umich.edu/",
```

Comment:
Maizey tell now support this new URL for Non-prod, hence I am changing it here. (But the Non-prod LTI configuration already configured for this)